azure-pipelines-pr.yml

```diff
@@ -29,6 +29,9 @@ jobs:
           filePath: yarn-ci.sh
         displayName: yarn (install packages)
 
+      - script: yarn playwright install --with-deps
```

Comment:
**FYI:** 

this is ok to live in wc3 branch. 

if this would be against master we would most likely not accept this, as the cost of prolonging pipeline just for sake of  1 package is not worth the slowdown. If whole monorepo moves to use playwright for all e2e in feature then it's totally valid. 

For this use-case we would need to make this on-demand as we do within fluent-contrib https://github.com/microsoft/fluentui-contrib/blob/main/packages/nx-plugin/src/executors/playwright/executor.ts#L51-L54

Reply:
How are the Playwright binaries currently made available for existing packages which use Storywright?